unionml/cli.py

```diff
@@ -50,11 +50,18 @@ def init(
 
 
 @app.command()
-def deploy(app: str):
+def deploy(
+    app: str,
+    ignore_diff: bool = typer.Option(False, "--ignore-diff", help="allow uncommitted changes"),
```

Comment:
`--ignore-diff` from the title of #56, but maybe `--allow-dirty`, `--untracked` or something else? Initially, I feel like `--ignore-diff` isn't as descriptive as I would like.

Reply:
how about `--allow-uncommitted` ? would also be down for `--allow-dirty`

Reply:
`--allow-uncommitted` is clear! I'll change to that